mantidproject/mantid
# Correcting naming and constraints in Indirect Data Analysis

**Description of work.**

This solved some of the issues listed in #28484, more specifically it corrects the issues related to work space naming and constraints. 

Constraints have been added to all the tabs and the work space naming has been improved in the Msd and FQ fit tabs.



**To test:**
  * Data to test the relevant tabs can be found in #26631
  * Do a git on the MSD and FQ tabs and check that the workspace names contain information about what sort of fit was done and make sense. 
  * Check that all fit functions on all tabs have correct constraints set.

Refs #28484  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
